packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -411,6 +411,7 @@ const DropDownEditor = TextBox.inherit({
         this._refreshValueChangeEvent();
         this._refreshEvents();
         this._refreshEmptinessEvent();
+        this.option('_onMarkupRendered')?.();
```

Comment:
The same note like here: https://github.com/DevExpress/DevExtreme/pull/27362/files#r1600917874